frontend/src/views/services.js

```diff
@@ -1,13 +1,23 @@
 const requiredFields = [
-  'date', 'car', 'description', 'from', 'destination', 'startMileage', 'endMileage',
+  'date',
+  'car',
+  'description',
```

Comment:
A remotely related thing: text fields are validated as correct when filled with spaces. This one we need to fix.